app/controllers/albums_controller.rb

```diff
@@ -0,0 +1,63 @@
+class AlbumsController < ApplicationController
```

Comment:
In the future you could consider using a filter with a before action.  

before_action :find_album, only: [:show, :edit, :upvote, :destroy]

...

private 
def find_album
  @myalbum = Album.find_by(params[:id].to_i)
end